bluesky/stack/stack.py

```diff
@@ -1508,7 +1509,7 @@ def parse(self, argtype, argidx, args):
 def distcalc(lat0, lon0, lat1, lon1):
     try:
         qdr, dist = geo.qdrdist(lat0, lon0, lat1, lon1)
-        return True, "QDR = %.2f deg, Dist = %.3f nm" % (qdr % 360., dist)
+        return bs.SIMPLE_ECHO, "QDR = %.2f deg, Dist = %.3f nm" % (qdr % 360., dist)
     except:
```

Comment:
^^ The output when echo'd does not need to repeat the command.  See `stack.process()`